storage/transfer/list.go

```diff
@@ -0,0 +1,11 @@
+package transfer
+
+import "github.com/Erikaa81/banco-erika/domain/entities"
+
+func (r Repository) List(string) ([]entities.Transfer, error) {
+	var transfersList []entities.Transfer
+	for _, transfer := range r.storage {
+		transfersList = append(transfersList, transfer)
+	}
+	return transfersList, nil
+}
```

Comment:
em complemento ao que foi falado sobre o método Create não estar fazendo nada, esse list também não vai listar nada já que o r.storage não está sendo preenchido

Reply:
além disso, a sintaxe esta incorreta aqui:
```suggestion
func (r Repository) List(accountID string) ([]entities.Transfer, error) {
	var transfersList []entities.Transfer
	for _, transfer := range r.storage {
		transfersList = append(transfersList, transfer)
	}
	return transfersList, nil
}
```

Reply:
ainda falta a questão de você filtrar as transferências feitas por determinada conta, mas esse ponto vou deixa pra ce matutar

Reply:
ok, vou corrigir e pensar nisso!